lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -480,7 +485,8 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
     def _delete_environment(self, environment: Environment, log: Logger) -> None:
         environment_context = get_environment_context(environment=environment)
         resource_group_name = environment_context.resource_group_name
-        assert resource_group_name
+        if not resource_group_name:
```

Comment:
Add comments here, how it happens. For example, the environment is not deployed.